R/get-path-from-object.R

```diff
@@ -47,6 +47,12 @@ get_model_path.bbi_nmboot_model <- function(.bbi_object, .check_exists = TRUE) {
   get_model_path_nonmem(.bbi_object, .check_exists)
 }
 
+#' @rdname get_path_from_object
+#' @export
+get_model_path.bbi_nmsim_model <- function(.bbi_object, .check_exists = TRUE) {
+  get_model_path_nonmem(.bbi_object, .check_exists)
```

Comment:
It looks like `get_model_path()`  works the same for most (all) objects? Do you want to make a `get_model_path.default()` method?

Reply:
Yeah that's a fair point. Was worried about `bbr.bayes` implications in some of the get_path helpers and just opted to copy-paste , but looking at the [method](https://github.com/metrumresearchgroup/bbr.bayes/blob/cc665a5a81ba51b9c11c160442c21361e025b6a7/R/get-path-from-object.R#L39), there isnt any concern for this function.